README.md

```diff
@@ -136,16 +135,19 @@ links: Ember.A([
 ```
 
 If you are using SASS, you must import the styles for the navbar into your styles.
+
 ```scss
 //app.scss
 
-//You can override the styles by setting the following variables:
+// You can override the styles by setting the following variables:
+$c-text-color: #F5F0E1; // cream
```

Comment:
In Sass, because any variable can be used anywhere, it's a convention to indicate what the variable is before it's name. In this case, `$c-` is color.

I was thinking, that it might be wise to namespace these variables in case the user is already using `$text-color` for something else. I would either suggest prepending `$e3dn-` before each var, or instead of individual variables, using a Sass map:

```scss
$ember-3d-nav-defaults: (
  'body-bg': #485274,
  'nav-container-bg': #000,
  'nav-bg': darken(map-get($ember-3d-nav, 'body-bg'), 10%),
  'text-color': #fff,
  'is-fixed-color': map-get($ember-3d-nav, 'nav-bg')),
  'nav-items': 5,
  'nav-height': 80px,
  'nav-height-medium': 170px,
  'animation-duration': 0.5s
);

$ember-3d-nav: () !default;

// Merges defaults with the empty $ember-3d-nav map or one supplied by the user
$ember-3d-nav: map-merge($ember-3d-nav-defaults, $ember-3d-nav);
```

Then, changing styles would be more like creating an options object. The user would just create a `$ember-3d-nav` map of some or all of the settings we have and could eventually document.